src/test/java/SetStudyTest.java

```diff
@@ -0,0 +1,34 @@
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class SetStudyTest {
+    private Set<Integer> numbers;
+
+    @BeforeEach
+    void setUp() {
+        numbers = new HashSet();
+        numbers.add(1);
+        numbers.add(1);
+        numbers.add(2);
+        numbers.add(3);
+    }
+
+    @Test
+    public void size() {
+        assertThat(numbers.size()).isEqualTo(3);
```

Comment:
다음과 같이도 테스트할 수 있을 것 같아요!

```suggestion
        assertThat(numbers).hasSize(3);
```